cli/crates/backend/assets/grafbase.default.config.ts

```diff
@@ -1,54 +1,40 @@
-import { g, auth, config } from '@grafbase/sdk'
+import { g, config } from '@grafbase/sdk'
 
 // Welcome to Grafbase!
-// Define your data models, integrate auth, permission rules, custom resolvers, search, and more with Grafbase.
-// Integrate Auth
-// https://grafbase.com/docs/auth
 //
-// const authProvider = auth.OpenIDConnect({
-//   issuer: process.env.ISSUER_URL ?? ''
-// })
-//
-// Define Data Models
-// https://grafbase.com/docs/database
-
-const post = g.model('Post', {
-  title: g.string(),
-  slug: g.string().unique(),
-  content: g.string().optional(),
-  publishedAt: g.datetime().optional(),
-  comments: g.relation(() => comment).optional().list().optional(),
-  likes: g.int().default(0),
-  tags: g.string().optional().list().length({ max: 5 }),
-  author: g.relation(() => user).optional()
-}).search()
-
-const comment = g.model('Comment', {
-  post: g.relation(post),
-  body: g.string(),
-  likes: g.int().default(0),
-  author: g.relation(() => user).optional()
-})
+// Configure authentication, data sources, resolvers and caching for your GraphQL API.
 
-const user = g.model('User', {
-  name: g.string(),
-  email: g.email().optional(),
-  posts: g.relation(post).optional().list(),
-  comments: g.relation(comment).optional().list()
+// Data Sources - https://grafbase.com/docs/connectors
+//
+// const pg = connector.Postgres('pg', { url: g.env('DATABASE_URL') })
+// g.datasource(pg)
 
-  // Extend models with resolvers
-  // https://grafbase.com/docs/edge-gateway/resolvers
-  // gravatar: g.url().resolver('user/gravatar')
+// Resolvers - https://grafbase.com/docs/resolvers
+//
+g.query('helloWorld', {
+  returns: g.string(),
+  resolver: 'hello-world',
```

Comment:
Kind of wonder if we should have a `hello-world` resolver in the default output so that this works...

Reply:
Yeah I thought about this too, but didn't want to include any resolvers. by default. I could either comment this one out, but then the schema is empty. The error will only happen when you query the field.